### DISCUSSION TITLE
STI Suite (version 3.0)

### DISCUSSION THREAD

**Discussion by milk50cc** [kschan0214/sepia#32] [General]

**Comment by ehm6** [comment]
Same here, unable to get link to STI Suite. Any alternative ideas? Is this toolbox required?
Thank you!

**Comment by kschan0214** [comment]
We've been in contact with the STI Suite team and it seems the issue is now solved. Perhaps you can try contacting them again to see if you can receive the download link. 

Besides, you can use SEPIA without certain toolboxes as long as you don't select the algorithms from these toolboxes. :)
